src/main/java/com/robertx22/mine_and_slash/a_libraries/dmg_number_particle/particle/GLUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.robertx22.mine_and_slash.a_libraries.dmg_number_particle.particle;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import org.lwjgl.opengl.GL11;
+
+public class GLUtils {
+
+    public static void renderAlwaysSeenText(Runnable render){
```

Comment:
What's the purpose of this? I see the name but were some particles not displaying? 
It seems like it could cause trouble with the render calls

Reply:
health bar will cover the particle.